### ISSUE TITLE
Show mentions in PRS/issues

### ISSUE THREAD

**Issue by danez** [slapperwan/gh4a#286]
It would be nice if one could see in the issue view when someone mentions this issue in an other one or a commit.

**Comment by maniac103** [comment]
We already do show references of an issue in a commit, but AFAICT from https://developer.github.com/v3/issues/events/ it doesn't look like there's an event defined for an issue being referenced from another issue. I'll check it though.

**Comment by maniac103** [comment]
I've checked this again, specifically with a NPM bug report [1]. In the middle of the report, one can see some PRs mentioning this issue (after the cvrebert commit). In the event stream [2] those appear as 'mention' without any way to trace those back to a PR. Also, e.g. the cvrebert commit (item number 23 in the JSON array) includes a commit ID, but doesn't include information on which repo this commit was added to.
AFAICT Octodroid doesn't have any chance of displaying those mentions the same way as Github's web UI does. I'll ask Github support, but from previous experience I'm not optimistic anything will change in the API any time soon (see #265 for that previous experience - they promised to fix that eventually for > 1 year now).

[1] https://github.com/npm/npm/issues/11283
[2] https://api.github.com/repos/npm/npm/issues/11283/events
